src/components/ConsoleIcons/index.tsx

```diff
@@ -132,7 +132,13 @@ export default function ConsoleLabel({name}): ReactElement {
     case 'users':
       return (
         <>
-          <ConsoleIconWrapper icon={ConsoleIcons.people} /> <b>Users</b>
+          <ConsoleIconWrapper icon={ConsoleIcons.user} /> <b>Users</b>
```

Comment:
That's not what I experience in the sidebar of any service?

<img width="92" alt="Screenshot 2024-05-13 at 12 42 33" src="https://github.com/aiven/aiven-docs/assets/28596581/03dc8085-581a-4bdf-b97c-70e81a0fe6cf">

Current state seems to be correct for that area. Where did you see the `users` label connected to a single user icon? We might need to add a case.

Reply:
Ah, okay, so this is service users? Org users are managed in Admin. 
![image](https://github.com/aiven/aiven-docs/assets/111294980/d7bfaf51-a493-4b93-af56-9c4dfc7e80ce)

Reply:
Speaking of which, I've just noticed as well that the view profile icon is different for org users versus applications users.

Reply:
Added cases for these 👍

Reply:
Missed the service users page, but asking the team about this since it doesn't seem totally accurate?